### PR TITLE
Promote Hab package alongside Omnibus package

### PR DESCRIPTION
**Pull request by schisamo**
This change ensures we are promoting our Hab packages to `current` and
`stable` at the same time we execute said promotions for the Omnibus
packages.

Signed-off-by: Seth Chisamore <schisamo@chef.io>

### CODE REVIEW

**Review by tduffield — CHANGES_REQUESTED** [review]
Blocked by https://github.com/chef/expeditor/issues/870

**Review by miah — APPROVED** [review]
Seems reasonable to me. Thanks @schisamo

**Review by tduffield — APPROVED** [review]

**Review by jerryaldrichiii — APPROVED** [review]
Looks good to me. Is this still blocked?

**Comment by jerryaldrichiii** [comment]
Once this is merged we should update our release instructions, since right now we do this manually.

**Pull request merged**
